biosimulations/apps/dispatch-api/src/app/app.controller.ts

```diff
@@ -105,7 +104,7 @@ export class AppController implements OnApplicationBootstrap {
     const simSpec: SimulationDispatchSpec = {
       authorEmail: bodyData.authorEmail,
       nameOfSimulation: bodyData.nameOfSimulation,
-      simulator: bodyData.simulator,
+      simulator: bodyData.simulator.toUpperCase(),
```

Comment:
The simulators at [biosimulators](http://biosimulators.org/simulators) will have all lowercase id's. We should be consistent between the two

Reply:
Just want to make sure this won't break anything on the backend. If not, I can make the change and then merge it in